lib/spack/docs/build_settings.rst

```diff
@@ -209,6 +209,170 @@ Specific limitations include:
   then Spack will not add a new external entry (``spack config blame packages``
   can help locate all external entries).
 
+
+.. _manually-adding-external-packages:
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Manually Adding External Packages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section discusses how to add external packages to the ``packages.yaml``
+file when modules depend on other modules or when Spack is unable to compute
+the prefix. When using modules, Spack will attempt to parse the ``module show``
```

Comment:
Overall I think this should be phrased differently (but it would take me significant time to come up with suggestions): our support for finding externals via modules is nonexistent (mentioned above in https://spack.readthedocs.io/en/latest/build_settings.html?highlight=external#automatically-find-external-packages). So I think this should emphasize that and overall that explain that this section describes how to resolve some of the resulting shortcomings.

As it is, this introduction names specific issues with using module-based externals so I think is too focused.